src/test/shared/schema/types.ts

```diff
@@ -0,0 +1,25 @@
+/*!
```

Comment:
This file can go in `schema/testUtil`.

Hyper-granularity harms discoverability by adding noise (which makes the signal harder to find). It's very unlikely that we will need these things in other test files--and if we ever do, it's easy to split things out (analysis is easier than synthesis).

Reply:
Yeah agreed, though I didn't find the organization much of an issue for this specific PR as most of the 'util' logic here will probably be extracted out and generalized.